packages/react-table/src/components/Table/TreeRowWrapper.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Table/table';
+import { RowWrapperProps } from './RowWrapper';
+import { Tr } from '../TableComposable';
+
+export const TreeRowWrapper: React.FunctionComponent<RowWrapperProps> = ({
+  className,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  rowProps,
```

Comment:
Is `rowProps` something used in non-composable tables? Why is it being withheld from spreading to the Tr?

Reply:
I am not totally sure all the reasons the rowData prop exists. At the very least it keeps track of rowIndex and rowKey for the sake of certain callback functions. But, I know that it's already being withheld in the RowWrapper, and if I don't withhold it in the TreeRowWrapper, it spreads all the way to the `<tr>` element and causes a console warning.